### PR TITLE
initial code commit

### PR DESCRIPTION
**Pull request by rodrigo-ldc**
First working and installable version =)

### CODE REVIEW

**Review by ymesquida — DISMISSED** [review]
SGTM

File: setup.py [hunk 1]
```diff
@@ -23,21 +23,21 @@ def read(fname):
     description='Map Server for Google Earth Engine',
     long_description=read('README.md'),
     url='',
-    author='Rodrigo E. Principe',
-    author_email='fitoprincipe82@gmail.com',
+    author='LDC Data Sciencie and Remote Sensing team (DSRS)',
```
Comment (ymesquida) [inline setup.py]: Sciencie > Science

Comment (ymesquida) [reply setup.py]: @rodrigo-ldc

**Review by ymesquida — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (rodrigo-ldc) [reply setup.py]: thanks =)

**Review by ymesquida — APPROVED** [review]

**Pull request merged**